xapian-letor/include/xapian-letor/feature.h

```diff
@@ -41,7 +41,7 @@ namespace Xapian {
 
 /// Abstract base class for features in learning to rank
 class XAPIAN_VISIBILITY_DEFAULT Feature {
-  protected:
+  public:
     /// Stats which FeatureList can use.
```

Comment:
The documentation comment for this enum now ought to say `@protected` so that the API docs continue to show it as protected.

Reply:
fixed in a fixup commit.